org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -508,6 +524,28 @@ package lang {
 		
 		method abs() native
 		method invert() native
+		method gcd(other) native
+		method lcm(other) {
+			const mcd = self.gcd(other)
+			return self * (other / mcd)
+		}
+		method digits() {
+			var digits = self.toString().size()
+			if (self < 0) {
+				digits -= 1
+			}
+			return digits
+		}
+		method prime() {
+			if (self == 1) return false
+			var isPrime = true
+			(2..self - 1).forEach({ i =>
+				// Horrible definition, but return doesn't exit from loop
+				// and I need fold, any methods in Range
```

Comment:
:+1:  It was very hard for me to implement this method.
First of all, I'd like to have fold, any, etc. methods in Range. 
Then, of course we should have a "real"return and a while circuit.